### PR TITLE
Update management template to Plone6 / Bootstrap 5

### PR DESCRIPTION
**Pull request by petschki**
I've removed `pat-livesearch` in favor of `pat-inject` to enable "live filtering" of the result list.

Before / After
Bildschirm­foto 2022-11-23 um 11 38 25

### CODE REVIEW

**Review by jensens — APPROVED** [review]
LGTM, may you branch away the current state for Plone 5?

File: src/Products/PloneKeywordManager/profiles/default/controlpanel.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     appId="PloneKeywordManager"
     category="Products"
     condition_expr=""
-    icon_expr="string:${portal_url}/book_icon.png"
+    icon_expr="string:tags"
```
[the remark this answers is not in the document]
Comment (jensens) [reply src/Products/PloneKeywordManager/profiles/default/controlpanel.xml:11]: would be favorable to have

**Comment by petschki** [comment]
I'll branch current master to `5.x` and would suggest to release "Plone 6 only changes" as `6.x` ... I will also remove deprecated code for the old tool and the no longer used `@@keywordsmanager-search-json` view here before merging. And of course an upgrade step for the actionicon 😉

**Comment by petschki** [comment]
I made some more changes notably a columnized listing when there are many keywords ...but for now I'm happy and will merge it.

Bildschirm­foto 2022-11-24 um 10 09 47

**Comment by petschki** [comment]
`column: 3` in CSS style gets a bit confused when showing the similar keywords

File: src/Products/PloneKeywordManager/tool.py [hunk 2]
```diff
@@ -163,19 +163,13 @@ def getScoredMatches(self, word, possibilities, num, score, context=None):
...
-            oword = word.encode("utf-8")
-
         for item in possibilities:
-            if isinstance(item, type(word)):
-                lscore = Levenshtein.ratio(word, item)
-            elif isinstance(item, type(oword)):
-                lscore = Levenshtein.ratio(oword, item)
+            if word.lower() in item.lower():
+                # if the word is a substring always include it
+                lscore = 1
```
Comment (petschki) [inline src/Products/PloneKeywordManager/tool.py:169]: @gforcada this is maybe also interesting for `5.x` updates. I got a too low score (Levenshtein or difflib) when the searchterm was a substring of a keyword but too short. I now do always add it when its a substring without checking the score.

Comment (gforcada) [reply src/Products/PloneKeywordManager/tool.py:169]: Oh, that's nice, I build a tag comparison export one day and I did the same, show both the `Levenshtein` distance as well as _who is inside who_ list as well.

In our case, we started with +50k tags, so having too low scores was not really a problem 😓 but I see the potential.

Hopefully within next year I end up working on this 🤞🏾

**Pull request merged**